### DISCUSSION TITLE
Please hide .terraform.lock.hcl file content in PR DIFF view by default

### DISCUSSION THREAD

**Discussion by nyurik** [community/community#5897] [Pull Requests] [answered]
Terraform has [version lock files](https://www.terraform.io/docs/language/dependency-lock.html) called `.terraform.lock.hcl`, the same way as `package-lock.json` for NodeJS. Please hide them by default in all the pull request diff views.  Thanks!

**Comment by nyurik** [comment, marked as answer]
Update: A PR has been submitted to fix this issue - https://github.com/github/linguist/pull/5610

**Reply by TAO92996my** [reply]
Hi
